frontend/src/components/dialog/DialogForm.vue

```diff
@@ -133,7 +133,7 @@ export default {
 }
 </script>
 
-<style lang="scss">
+<style lang="scss" scoped>
 @media #{map-get($display-breakpoints, 'xs-only')} {
   .ec-dialog-form {
```

Comment:
.ec-dialog-form seems also referenced in other components (`DetailPane`, `DialogBottomSheet`). We might need to double check if it's really needed in these other components.